guide/how-it-works/private-key-management/multi-key.md

```diff
@@ -74,7 +76,57 @@ A software wallet application or coordination software initiates a multi-sig wal
 
 {% include fact/close.html %}
 
-### Best practice
+### General user flow
+
+Alice, Robert, and Charles want to set up a multi-key wallet that has a total of 3 keys, with 2 keys required to send a transaction.
+
+In the first step, each of them independently generate a private key and a public key (technically an extended public key). They will use the private key for initiating and approving transactions, and will never share it it. The public key is used for construct the multi-key wallet for generating addresses to receive bitcoin to.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-generation.png"
+   retina = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-generation@2x.png"
+   mobile = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-generation-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-generation-mobile@2x.png"
+   alt-text = "Diagram of Alice, Robert, and Charles generating private and public keys"
+   width = 800
+   height = 456
+%}
+
+Then, they exchange the public keys, so each of them has a set of all 3.
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-exchange.png"
+   retina = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-exchange@2x.png"
+   mobile = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-exchange-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/private-key-management/multi-key/multi-key-diagram-1-key-exchange-mobile@2x.png"
+   alt-text = "Diagram of the three wallet owners exchanging public keys"
+   width = 800
+   height = 381
+%}
+
+They can now individually construct the same multi-key wallet with the 3 public keys and the spending conditions they decided on (2-of-3). Addresses generated by each wallet will be identical. The order of the public keys is important here. A different order will result in a different wallet.
```

Comment:
The order of the public keys can be ensured by the software, so it is not necessary for the signers themselves to sign in any particular order. Unsure if this necessitates any change to the text here.

Reply:
I think this would be a good addition.

Reply:
The order has to be the same for constructing the wallet, but not for signing. I tested this with the [BitcoinJS library](https://github.com/bitcoinjs/bitcoinjs-lib) earlier this week.

In the wallet coordination file, there are fingerprints of each key. That's how the order can be tracked later once the wallet is set up.

Reply:
I added an extra note about this.